src/bootstrap/flags.rs

```diff
@@ -75,7 +75,22 @@ pub enum Subcommand {
 
 impl Flags {
     pub fn parse(args: &[String]) -> Flags {
+        let mut extra_help = String::new();
```

Comment:
No worries.  It is mentioned in both the commit message where the change was made (but not on the first line of the message) and in the PR text.  I also stated in the PR text that this resolves #38373, which is a request to accept positional arguments after options.

If there is something I can do to make it more clear, I'd be happy to.  Just let me know.  Should I amend the commit it was in to get that info into the first commit line?